File: f/src/types/UserOrder.ts

type UserOrder = {
  transaction_id?: number
  cart_id?: number,
  quantity?: number,
  product_id?: number,
  product_name?: string,
  product_image?: string
  product_price?: number,
  uploaded_by?: number,
  delivery_status?: string
}

export default UserOrder

File: b/model/UserOrder.go
package model

import "gorm.io/gorm"

type UserOrder struct {
	gorm.Model
	TransactionId  uint   `json:"transaction_id"`
	CartId         uint   `json:"cart_id"`
	Quantity       uint   `json:"quantity"`
	ProductId      uint   `json:"product_id"`
	ProductName    string `json:"product_name"`
	ProductImage   string `json:"product_image"`
	ProductPrice   uint   `json:"product_price"`
	UploadedBy     uint   `json:"uploaded_by"`
	DeliveryStatus string `json:"delivery_status"`
}
